File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.cs
using System;

public class Solution 
{
    public string solution(string my_string, string letter) 
    {
        string result = "";
        for(int i=0; i<my_string.Length; i++)
        {
            if(my_string[i].ToString() != letter)
            {
                result += my_string[i].ToString(); 
            }
        }
        return result;
    }
}

File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, string letter) 
{
    while(my_string.find(letter[0]) != string::npos)
    {
        int pos = my_string.find(letter[0]);
        my_string.erase(pos,1);
    }
    return my_string;
}